Tanja Stroble

```diff
@@ -0,0 +1,73 @@
+########################################################
+# Step 1: Establish the layers
+
+# Write a list of the layers here
+# I set up an array named "drivers"
+# The indices of drivers array are driver1 - driver4, all hashes containing
+# the :keys ride_id, pay, rating,and dates_active.
+# corresponding elements were fed into an array, making the
+# data structure an array of hashes with arrays as values.
+#
+########################################################
+# Step 2: Assign a data structure to each layer
+# top layer - array (4 elements)
+# 2nd layer - hash (4 elements containing inside array of up to 3 elements)
+
+# Copy your list from above, and write what data structure each layer should have
+# top layer - array (4 elements) - should contain hash
+# 2nd layer - hash -  4 elements containing a symbol as key and  array of up to 3 elements as value)
+# values are arrays containing strings and integers.
+########################################################
+# Step 3: Make the data structure!
+
+# Setup the data strcture and manually write in data presented in rides.csv
+
+########################################################
+# Step 4: Total Divers Earnings and Number of Rides
+
+# Use an iteration block to print driver's total rides and money made
+
+
+i = 0
+drivers = [ # drivers array
+  #index 0 of drivers array
+  driver1 = {
```

Comment:
You don't need things like this `driver1 = {` here.  

I also think making these internal hashes containing 3 arrays, rather awkward to traverse.  It's also problematic if you accidentally add an element to one array and not the others, they become out of synch.  

I would suggest instead doing something like:
```ruby
driver = [
  {RD0073: [
    {ride_id: "RD0003", rating: 3, pay: 10, dates_active: "2016-02-03"},
    {ride_id: "RD0004", rating: 4, pay: 12, dates_active: "2016-02-07"}
    ]
  }
]    
```

That makes it easier to loop through things and keeps trips together logically.